Source/DafnyLanguageServer/Language/Symbols/ScopeSymbol.cs

```diff
@@ -1,15 +1,19 @@
 ﻿using System.Collections.Generic;
 using System.Threading;
+using Microsoft.Boogie;
 
 namespace Microsoft.Dafny.LanguageServer.Language.Symbols {
   public class ScopeSymbol : Symbol, ILocalizableSymbol {
-    public BlockStmt Declaration { get; }
-    public object Node => Declaration;
-    public List<ISymbol> Symbols { get; } = new List<ISymbol>();
+    public object Node { get; }
+    public readonly IToken Tok;
```

Comment:
This looks way better now, thanks for refactoring.
Would you mind giving the tokens more expressive names, e.g., `BodyStartToken` and `BodyEndToken`?